stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1225,6 +1234,12 @@ public final class TestSuite {
       f()
     }
     let test = _testByName(testName)
+
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
+    let nametag = name
+    startTrackingObjects(UnsafeMutablePointer(nametag._core.startASCII))
```

Comment:
@gribozavr while it should be safe to use a closure here, I would rather that we memcpy into an UnsafeMutablePointer that is allocated statically to ensure 100% that we do not have object allocation/deallocation.